ksy_schema.json

```diff
@@ -216,7 +224,13 @@
             }
           }
         },
-        "license": { "type": "string" },
+        "tags": {
+          "description": "any data that can give additional information about format. Should be used only at top-level"
```

Comment:
This isn't a good description of `tags` IMO. `tags` can't contain "any data" - that's what `meta` is for. `tags` has to be a list of strings (although the compiler doesn't check this), and each of them should look like an identifier. It would be nice if we had a list of currently used tags that we could link to here, to give people an idea about what sort of things should go into `tags`.

Reply:
Yes, I also found this strange, but the [corresponding git commit](421e9363d4179e96124c4a590291bc96faeb6671) said only about arbitrary content, so I didn't specify. Will change

Reply:
Fixed link to the commit: kaitai-io/kaitai_struct_compiler@421e9363d4179e96124c4a590291bc96faeb6671

The message says "ksc ignores the contents", but that's just because ksc itself doesn't do anything with it. ksc also ignores the contents of `meta/xref`, `meta/application`, etc., but that doesn't mean you should put arbitrary data in them 🙂

> And we have such examples? I didn't see any...

@GreyCat has been adding tags to many specs in kaitai_struct_formats recently, see for example kaitai-io/kaitai_struct_formats@91a7595fc27dea32b9c6b05b08414c557b948700.

Reply:
Ah. I have tried to search in my local copy but forgot to update it from project-super-pointer of `kaitai_struct` repo to `master` of `formats` repo :). Add a commit with "examples" key